test/e2e/specs/editor/various/__snapshots__/splitting-and-merging-blocks-test-restore-sele-2a1ee-roduces-more-than-one-block-on-forward-delete-1-chromium.txt

```diff
@@ -1,3 +1,13 @@
 <!-- wp:paragraph -->
-<p>hi-</p>
-<!-- /wp:paragraph -->
\ No newline at end of file
+<p>hi</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p>item 1</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:list -->
+<ul><!-- wp:list-item -->
+<li>item 2</li>
+<!-- /wp:list-item --></ul>
+<!-- /wp:list -->
```

Comment:
This looks different than the original Puppeteer [snapshot](https://github.com/WordPress/gutenberg/pull/44916/files#diff-aa89768efbf2cfe842d289dd72f56f3b42377ea01c97c121103d30e45d81b59eL241-L251). Is this expected result?


```
<!-- wp:paragraph -->
<p>hi</p>
<!-- /wp:paragraph -->
<!-- wp:paragraph -->
<p>item 1</p>
<!-- /wp:paragraph -->
<!-- wp:paragraph -->
<p>item 2</p>
<!-- /wp:paragraph -->
```